industrial_ci/src/env.sh

```diff
@@ -123,6 +123,10 @@ function set_ros_variables {
     "eloquent")
         ros2_defaults "bionic"
         ;;
+    "foxy")
+        ros2_defaults "bionic"
+        DOCKER_IMAGE=osrf/ros2:nightly
```

Comment:
```suggestion
        DOCKER_IMAGE=${DOCKER_IMAGE:-osrf/ros2:nightly}
```
This allow sderived images as well.

Reply:
Thanks!